.idea/gradle.xml

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
```

Comment:
Please remove this file from the changes and add this file to the gitignore too 🙏